src/main/java/baritone/process/MineProcess.java

```diff
@@ -111,7 +111,7 @@ public PathingCommand onTick(boolean calcFailed, boolean isSafeToCancel) {
         int mineGoalUpdateInterval = Baritone.settings().mineGoalUpdateInterval.value;
         List<BlockPos> curr = new ArrayList<>(knownOreLocations);
         if (mineGoalUpdateInterval != 0 && tickCount++ % mineGoalUpdateInterval == 0) { // big brain
-            CalculationContext context = new CalculationContext(baritone, true);
+            CalculationContext context = new CalculationContext(baritone, true, null); // Precomputed data should never be used on this calculation context
```

Comment:
Actually `rescan` calls `searchWorld` which calls `prune` which calls `plausibleToBreak` which calls `getMiningDurationTicks`, which SHOULD call the precomputed `canWalkThrough` (but doesn't currently)